packages/headless/src/features/insight-user-actions/insight-user-actions-slice.ts

```diff
@@ -0,0 +1,32 @@
+import {createReducer} from '@reduxjs/toolkit';
+import {
+  fetchUserActions,
+  registerUserActions,
+} from './insight-user-actions-actions';
+import {getInsightUserActionsInitialState} from './insight-user-actions-state';
+
+export const insightUserActionsReducer = createReducer(
+  getInsightUserActionsInitialState(),
+  (builder) => {
+    builder
+      .addCase(registerUserActions, (state, action) => {
+        state.ticketCreationDate = action.payload.ticketCreationDate;
+        if (action.payload.excludedCustomActions) {
+          state.excludedCustomActions = action.payload.excludedCustomActions;
+        }
+      })
+      .addCase(fetchUserActions.pending, (state) => {
+        state.loading = true;
+        state.error = undefined;
+      })
+      .addCase(fetchUserActions.rejected, (state, action) => {
+        state.loading = false;
+        state.error = action.payload;
+      })
+      .addCase(fetchUserActions.fulfilled, (state, _action) => {
```

Comment:
```suggestion
      .addCase(fetchUserActions.fulfilled, (state, _action) => {
```
```suggestion
      .addCase(fetchUserActions.fulfilled, (state, action) => {
```

Reply:
This is made on purpose actually cause for now the parameter `action` is not used in the function, so we need name it `_action`  to avoid having an error saying that we have an unused variable.

When you will add your part and you will start using this parameter we can update the name to be `action`.